deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -2856,7 +2856,7 @@ spec:
                       type: string
                       x-kubernetes-validations:
                         - message: network provider must be disabled (reverted to empty string) before a new provider is enabled
-                          rule: self == '' || self == oldSelf
+                          rule: oldSelf == '' || self == oldSelf
```

Comment:
@subhamkrai @BlaineEXE Do we need both `self` and `oldSelf` to be allowed to be blank to convert either direction?
```suggestion
                          rule: self == '' || oldSelf == '' || self == oldSelf
```

Reply:
That's what I was thinking as well

Reply:
Thanks for the review.
Good catch, that makes sense, let me fix that.